client/ctclient/ctclient.go

```diff
@@ -217,6 +220,66 @@ func getInclusionProofForHash(ctx context.Context, logClient *client.LogClient,
 	}
 }
 
+func getConsistencyProof(ctx context.Context, logClient *client.LogClient) {
+	if *treeSize <= 0 {
+		log.Fatal("No valid --size supplied")
+	}
+	if *prevSize <= 0 {
+		log.Fatal("No valid --prev_size supplied")
+	}
+	var hash1, hash2 []byte
+	if *prevHash != "" {
+		var err error
+		hash1, err = hex.DecodeString(*prevHash)
+		if err != nil {
+			log.Fatalf("Invalid --prev_hash: %v", err)
+		}
+		if l := len(hash1); l != sha256.Size {
+			log.Fatalf("Invalid --prev_hash length %d", l)
```

Comment:
nit: "length: %d" to match message above? same below?

Reply:
Done.